### PR TITLE
Expand Abstract and add Acknowledgements.

### PR DESCRIPTION
**Pull request by bencebeky**

### CODE REVIEW

**Review by birenroy — APPROVED** [review]

File: draft-beky-httpbis-metadata.md [hunk 1]
```diff
@@ -233,4 +236,5 @@ This document adds an entry to the "HTTP/3 Frame Types" registry defined in
 # Acknowledgments
 {:numbered="false"}
 
-TODO acknowledge.
+The authors would like to acknowledge Dianna Hu and Ian Swett for their
```
Comment (diannahu) [inline draft-beky-httpbis-metadata.md:239]: :)

File: draft-beky-httpbis-metadata.md [hunk 2]
```diff
@@ -31,7 +31,10 @@ author:
 --- abstract
 
 This document describes a mechanism to send meta information over HTTP/2
-({{!RFC9113}}) and HTTP/3 ({{!RFC9114}}) connections.  TODO elaborate
+({{!RFC9113}}) and HTTP/3 ({{!RFC9114}}) connections that refers either to the
+entire connection or a specific stream without changing the semantics of the
```
Comment (diannahu) [inline draft-beky-httpbis-metadata.md:35]: uber-nit: "entire connection or to a specific" ("to" for grammatical parallelism).

maybe better: "refers to either the entire connection or a specific stream" (also has parallelism).

**Review by diannahu — APPROVED** [review]
:)

File: draft-beky-httpbis-metadata.md (hunk 2 above)
Comment (bencebeky) [reply draft-beky-httpbis-metadata.md:35]: Oh nice, I always liked parallelism, and now I learned its name!

**Comment by bencebeky** [comment]
Thank you both for the review.

**Pull request merged**